docs/readthedocs/source/doc/DLlib/Overview/keras-api.md

```diff
@@ -2831,3 +2831,45 @@ array([[[[0.43790358, 0.61913717, 0.2543214 ],
         [[0.12074634, 0.6571231 , 0.752728  ],
          [0.86969995, 0.6700518 , 0.36353552]]]], dtype=float32)
 ```
+---
+## 8. Persistence
+This section describes how to save and load the Keras-like API.
+
+### 8.1 save
+To save a Keras model, you call the method `saveModel(path)`.
+
+**Scala:**
+```scala
+import com.intel.analytics.bigdl.dllib.keras.layers.{Dense, Activation}
+import com.intel.analytics.bigdl.dllib.keras.models.Sequential
+
+val model = Sequential[Float]()
+model.add(Dense[Float](32, inputShape = Shape(128)))
+model.add(Activation[Float]("relu"))
+model.saveModel("/tmp/seq.model")
+```
+**Python:**
+```python
+import bigdl.dllib.keras.Sequential
+from bigdl.dllib.keras.layer import Dense
+
+model = Sequential()
+model.add(Dense(input_shape=(32, )))
+model.saveModel("/tmp/seq.model")
+```
+
+### 8.2 load
+To load a saved Keras model, you call the method `load(path)`.
+
+**Scala:**
+```scala
+import com.intel.analytics.bigdl.dllib.keras.Net
+
+val model = Net.load[Float]("/tmp/seq.model")
+```
+
+**Python:**
+```python
+from bigdl.dllib.net import Net
+Net.load("/tmp/seq.model")
+```
```

Comment:
Need to make it consistent with Keras (https://www.tensorflow.org/api_docs/python/tf/keras/models/load_model, https://www.tensorflow.org/api_docs/python/tf/keras/models/save_model):

```python

model.save_model(...)

from bigdl.dllib.keras.models import load_model
model = keras.models.load_model("/tmp/seq.model")
```

```scala

model.saveModel(...)

import bigdl.dllib.keras.Models
model = Models.load_model("/tmp/seq.model")
```